react-frontend/src/components/StyleComponents/htmlTags.js

```diff
@@ -21,7 +21,7 @@ export const BreadcrumbUL = styled.ul`
   display: flex;
   flex-direction: row;
   list-style: none;
-  margin: 0;
+  margin: 0 0 0 -8px; /* negative left margin aligns breadcrumb with page elements */
```

Comment:
I'm going to push back hard on anything messing with the left alignment.   The entire point of the grid refactor was not to have margin-left and padding left things like this all over the codebase

Reply:
Is there a different way you would suggest to get the breadcrumb Home and navbar Home aligned?

Reply:
I just pushed one up on the grid pr (in the breadcrumb style)  This is a case where I think inline styling makes sense.  I just set paddingLeft:0 on the first element.